app/scripts/components/projectPanel/projectPanel.tsx

```diff
@@ -9,7 +9,7 @@ export const ProjectPanel = ({
                                counter = DEFAULT_PROJECT_PANEL_COUNT,
                                projectName = DEFAULT_PROJECT_PANEL_NAME
                              }: {
-  counter: string;
+  counter: string | number;
```

Comment:
I don't like the idea of string | number type. I think counter should be just number. If we have it somehow as string (from inputs etc), then it will be better to convert it to number later.